src/victory-animation/util.js

```diff
@@ -22,7 +21,7 @@ export const isInterpolatable = function (obj) {
       return false;
     case "object":
       // Don't try to interpolate class instances (except Date or Array).
-      return _.isDate(obj) || _.isArray(obj) || _.isPlainObject(obj);
+      return obj instanceof Date || Array.isArray(obj) || typeof obj === "object";
```

Comment:
This is not equivalent for `isPlainObject` if we really need that.

https://lodash.com/docs#isPlainObject

```
function Foo() {
  this.a = 1;
}

_.isPlainObject(new Foo);
// → false

typeof new Foo === "object"
// → true
```